src/GitHub.Api/Git/Tasks/GitCommitTask.cs

```diff
@@ -8,17 +8,15 @@ class GitCommitTask : ProcessTask<string>
         private const string TaskName = "git commit";
         private readonly string arguments;
 
-        public GitCommitTask(string message, string body,
+        public GitCommitTask(string messageFile,
             CancellationToken token, IOutputProcessor<string> processor = null)
             : base(token, processor ?? new SimpleOutputProcessor())
         {
-            Guard.ArgumentNotNullOrWhiteSpace(message, "message");
+            Guard.ArgumentNotNullOrWhiteSpace(messageFile, "messageFile");
```

Comment:
You can use `nameof(messageFile)` here so the compiler helps check against potential regressions when renaming things